style.css

```diff
@@ -0,0 +1,46 @@
+* {
```

Comment:
This file is autogenerated. You should add every autogenerated file to gitignore